### ISSUE TITLE
Add the ability to hold a stone in hand

### ISSUE THREAD

**Issue by 2ee4jnufew** [studio-minus/ppg-bugs-and-requests#1583]
**Feature description**
Add the ability to hold a stone in your hand.

**Feature purpose**
Add the ability to hold a stone in your hand, or make the stone smaller and make it possible to hold it in your hand.

**Additional context**
![unkn1own](https://user-images.githubusercontent.com/69415642/176962271-1696eac0-4f2a-4099-8033-86d65f6a2bd7.png)

Or

![Скриншот 01-07-2022 231349](https://user-images.githubusercontent.com/69415642/176962766-33a17556-21e6-43a3-993b-e413d3b28c72.jpg)

**Comment by DindinYT37** [comment]
I agree but disagree at the same time. It may be good since like, you can hold stones - but then it might also be bad since a human could be grabbing a background or scenery instead of the item he's supposed to (we all know how wonky the grabbing mechanism of a living being can be).
